src/elements/table/Table.js

```diff
@@ -259,8 +259,8 @@ export class NovoTable {
      * @param filter
      */
     onFilterClick(column, filter) {
-        if (Array.isArray(column.filter)) {
-            if (column.filter.includes(filter)) {
+        if (Array.isArray(column.filter) && column.multiple) {
```

Comment:
Just tested it and it does change the current functionality for selection based filters. Check the Table -> Basic -> Filter Status Column. This was multiple by default.

I am fine with making it single by default and making the `multiple=true`, but you would have to go through all the columns that are currently inside all Novo apps and make sure they get `multiple=true` if they were before.

@krsween @bvkimball thoughts?